From 102c046c7f5a0d5c7f88d91c3811b0c44772db29 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Fri, 5 Oct 2018 00:09:40 +0300
Subject: [PATCH 03/17] legacy: Resolve hal_bluetooth_default denial

* This comes up when switching bluetooth to hwbinder
* Mimic the old bluetooth rule to resolve the denial

Change-Id: I4852e3efc413061a4fc459259395ff9586605c12
---
 legacy-common/hal_bluetooth_default.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 legacy-common/hal_bluetooth_default.te

diff --git a/legacy-common/hal_bluetooth_default.te b/legacy-common/hal_bluetooth_default.te
new file mode 100644
index 0000000..aef3417
--- /dev/null
+++ b/legacy-common/hal_bluetooth_default.te
@@ -0,0 +1 @@
+allow hal_bluetooth_default wcnss_filter:unix_stream_socket connectto;
-- 
2.17.1

